core/src/main/java/org/apache/accumulo/core/spi/scan/DefaultScanServerDispatcher.java

```diff
@@ -150,28 +324,27 @@ public Duration getBusyTimeout() {
 
     Map<TabletId,String> serversToUse = new HashMap<>();
 
-    long maxBusyAttempts = 0;
+    // get the max number of busy attempts, treat errors as busy attempts
+    int attempts = params.getTablets().stream()
+        .mapToInt(tablet -> params.getAttempts(tablet).size()).max().orElse(0);
 
-    for (TabletId tablet : params.getTablets()) {
+    String scanType = params.getHints().get("scan_type");
 
-      // this a count of errors and busy attempts, will treat errors as busy
-      long busyAttempts = params.getAttempts(tablet).size();
+    Profile profile = null;
 
-      maxBusyAttempts = Math.max(maxBusyAttempts, busyAttempts);
+    if (scanType != null) {
+      profile = profiles.getOrDefault(scanType, defaultProfile);
```

Comment:
I'm thinking that if the application specifies a profile name, but it doesn't exist, then an error should be thrown?